Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -186,6 +259,54 @@
         "CannotFindBuilds": "Cannot find builds",
         "JenkinsNoCommitsToFetch": "Start and end build Ids are same. Commits and workitems will not be downloaded.",
         "InvalidJenkinsBuildNumber": "Cannot parse the Jenkins Build number.",
-        "InvalidJenkinsStartBuildNumber": "Invalid value mentioned for Jenkins start build number."
+        "InvalidJenkinsStartBuildNumber": "Invalid value mentioned for Jenkins start build number.",
+        "ClientIdCannotBeEmpty": "clientId must be a non empty string.",
```

Comment:
For all "must be a non empty string" -> recommend something like "ClientID has to be specified". Null and non-empty are implementation detail which we don't want to show in error strings.